app/Http/Controllers/Api/V1/MicroSoftTeamController.php

```diff
@@ -211,6 +251,35 @@ public function getClasses(Request $request)
         ], 200);
     }
 
+    /**
+     * Get User profile
+     *
+     * Get User profile of Microsoft Team
```

Comment:
Doc blocks need to update to standard, such as for the params required by GetUserProfileRequest

Please consult following message by Waleed.
https://currikiworkspace.slack.com/archives/C01UQKSFDGB/p1668421603000099